tests/test0000.d

```diff
@@ -1,3 +1,5 @@
+// RUN: %sdc %s -o %t && sh -c '%t; [ $? = 42 ]'
```

Comment:
An advantage of using lit is that you can easily check for stdout too (for the tests that are not supposed to compile). Imho, lit is basically an improved runner.d that SDC does not need to maintain.
At the moment, runner.d is running SDC again and again. Were you planning on changing runner.d?